webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Overview/styles.css

```diff
@@ -5,9 +5,3 @@
 .details-cards {
   padding-top: 20px;
 }
-
-.details-tab .pf-c-accordion {
-  height: 15vh;
-  overflow-y: auto;
-  overflow-x: hidden;
-}
```

Comment:
This was used for the audits card when it was an accordion https://github.com/theforeman/foreman/pull/8598